ansible/roles/dev/tasks/main.yml

```diff
@@ -97,16 +98,29 @@
       append: true
 
 - name: Install Pulp RPM dependencies
-  dnf:
+  yum:
       state: present
       name: "{{ pulp_rpm_dependencies }}"
 
 - name: Ensure homedir mode suitable for ssh keys
-  file: dest=/home/{{ ansible_env.SUDO_USER }} mode=0750
```

Comment:
It's been a while (months) since I made these changes but IIRC it was actually complaining or warning about this, it wasn't just an aesthetic formatting change.

Reply:
Or maybe it was the other standalone items without a name below, and I just changed others for consistency.  I don't remember anymore.